Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -570,6 +580,11 @@ def set_data(self, data):
         else:
             self.invalidate(unconditional=True)
 
+    def _set_input_summary(self):
```

Comment:
Just a question. Why the `input` has a function and the `output` hasn't?

Reply:
I overlooked this, I will add a function for the `output` now.